developer-docs-site/docs/guides/building-your-own-wallet.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Building Your Own Wallet"
+slug: "building-your-own-wallet"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building Your Own Wallet
+
+There are already a few wallets building on Aptos. To allow for easy wallet interoperability we need to do the same things across wallets:
```

Comment:
In order to allow for Aptos' wallet interoperability, the following is required: